Tests/DiligentToolsTest/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required (VERSION 3.6)
 project(DiligentToolsTest)
 
 file(GLOB_RECURSE INCLUDE include/*.*)
-file(GLOB_RECURSE SOURCE src/RenderStateNotationParser/*.*)
```

Comment:
You should simply use src/*.*  - you are still missing RenderStatePackager folder